Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -1003,6 +1005,10 @@ void UpdateToolbar()
 			if (!textColor.IsDefault)
 				bar.SetTitleTextColor(textColor.ToAndroid().ToArgb());
 
+			Color navIconColor = NavigationPage.GetIconColor(Current);
+			if(!navIconColor.IsDefault)
```

Comment:
Looks like you won't be able to set the color back to default after setting it to an explicit color.

Reply:
but it's a generic issue of all platform-specific stuff.

CursorColorProperty from iOS-specific Entry has exactly the same issue. (and a lof of other)

Reply:
We miss it a lot, sure. We have an opportunity to not miss it here! The pattern we try to use is to cache the first value that was already set in a field (like `_defaultNavIconColor`); if the navIconColor is then set to `Default`, we set it to the field's value.

Reply:
@samhouts makes sense) but how would we handle cases, when a user changes something via custom renderer (properties in controls directly). What is "real" default value?

Reply:
By the time this code is run, we hope that the custom renderer has already set the value of the color filter, so whatever we get from the current color filter is what they've set....theoretically! That's why I try to always test new changes against a custom renderer that's already set the value I'm affecting.

Reply:
@samhouts great. now is clear) thx